BattleBump/Player.swift

```diff
@@ -7,37 +7,27 @@
 //
 
 import UIKit
+import MultipeerConnectivity
 
-class Player: NSObject, NSCoding {
-
-    @objc let name: String
-    @objc let emoji: String
-    @objc var move: String
-
-    @objc init(name: String, emoji:String, move: String) {
-        
-        self.name = name
-        self.emoji = emoji
-        self.move = move
-        super.init()
-    }
-   
-    // MARK: NSCoding
+class Player: Codable {
     
-    required convenience init?(coder decoder: NSCoder) {
-        
-        guard let name = decoder.decodeObject(forKey: "name") as? String,
-            let emoji = decoder.decodeObject(forKey: "emoji") as? String,
-            let move = decoder.decodeObject(forKey: "move") as? String
-            else { return nil }
-        
-        self.init(name: name, emoji: emoji, move: move)
-    }
+    var name: String
+    var selectedMove: String // TODO: change to use Moveset.swift etc
```

Comment:
Best practice would be to declare this optional, and not use `""` the empty string to indicate no selected move, but rather `nil`